example/package.json

```diff
@@ -70,6 +70,7 @@
     "@cumulus/test-processing": "18.1.0"
   },
   "devDependencies": {
+    "@aws-sdk/client-ecs": "^3.485.0",
```

Comment:
We should probably use the same version as the other V3 dependencies: `^3.447.0`